### ISSUE TITLE
az image copy failed with error

### ISSUE THREAD

**Issue by vinaykumar-rm** [Azure/azure-cli-extensions#5258]
## Describe the bug

**Command Name**
`az image copy
Extension Name: image-copy-extension. Version: 0.2.9.`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
expected str, bytes or os.PathLike object, not NoneType
Traceback (most recent call last):
  File "/opt/az/lib/python3.10/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 663, in execute
    raise ex
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 697, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 333, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/root/.azure/cliextensions/image-copy-extension/azext_imagecopy/custom.py", line 108, in imagecopy
    run_cli_command(cli_cmd)
  File "/root/.azure/cliextensions/image-copy-extension/azext_imagecopy/cli_utils.py", line 21, in run_cli_command
    cmd_output = check_output(cmd, stderr=STDOUT, universal_newlines=True)
  File "/opt/az/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/az/lib/python3.10/subprocess.py", line 501, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/opt/az/lib/python3.10/subprocess.py", line 969, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/az/lib/python3.10/subprocess.py", line 1778, in _execute_child
    self.pid = _posixsubprocess.fork_exec(
TypeError: expected str, bytes or os.PathLike object, not NoneType

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az image copy --source-resource-group {} --source-object-name {} --target-location {} --target-resource-group {} --source-type {}`

## Environment Summary
```
Linux-5.13.0-30-generic-x86_64-with-glibc2.31, Ubuntu 20.04.4 LTS
Python 3.10.5
Installer: DEB

azure-cli 2.39.0

Extensions:
image-copy-extension 0.2.9

Dependencies:
msal 1.18.0b1
azure-mgmt-resource 21.1.0b1
```

**Comment by yonzhan** [comment]
@zhoxing-ms for awareness

**Comment by zhoxing-ms** [comment]
@vinaykumar-rm Could you please send me the debug log *(Add `--debug` parameter)* by email? My email address is Zhou.Xing@microsoft.com, thanks~

**Comment by yanzhudd** [comment]
Hello @vinaykumar-rm, we fix some relevant content. You can use a private package in blow link to check if it meets your expectations.

https://zhuyan.blob.core.windows.net/zhuyan-container/image_copy_extension-0.2.10-py2.py3-none-any.whl

**Comment by balgaly** [comment]
hey @yanzhudd, i tried to use your package in Azure Shell and got this error:
`Request to https://zhuyan.blob.core.windows.net/zhuyan-container/image_copy_extension-0.2.10-py2.py3-none-any.whl failed with 409`

**Comment by yanzhudd** [comment]
> hey @yanzhudd, i tried to use your package in Azure Shell and got this error: `Request to https://zhuyan.blob.core.windows.net/zhuyan-container/image_copy_extension-0.2.10-py2.py3-none-any.whl failed with 409`

Hi @balgaly, you could retry by this command:
az extension add --source https://privatepackage.blob.core.windows.net/extension/image_copy_extension-0.2.10-py2.py3-none-any.whl --yes

**Comment by balgaly** [comment]
Thanks a lot @yanzhudd, any chance you know when this fix will be merged and be part of image-copy-extension?

**Comment by yanzhudd** [comment]
> Thanks a lot @yanzhudd, any chance you know when this fix will be merged and be part of image-copy-extension?

Hi @balgaly, if the private package could meet your expectation, we would merge it into a part of image-copy-extension. So please let us know if this package goes well or has any issues.

**Comment by balgaly** [comment]
The described problem did not reoccur after I used your private package.

**Comment by yanzhudd** [comment]
Hi @balgaly, this fix has merged and a new version, **0.2.10**, of image-copy-extension has released. Please use this command to upgrade the version:
az extension update --name image-copy-extension
